src/main/java/uk/gov/hmcts/reform/em/stitching/pdf/PDFCoversheetService.java

```diff
@@ -25,6 +25,9 @@
         addText(document, coversheet, pair.getFirst().getDocDescription(), 50,80, PDType1Font.HELVETICA_BOLD,13);
         moveLastPageToFirst(document);
 
+        PDFOutlineService pdfOutlineService = new PDFOutlineService();
```

Comment:
Probably best to set this up as a dependency (class property) rather than creating your own. DI makes it easier to test

Reply:
Dl? I'm gonna guess dependency injection.

Reply:
After second reading I think you should take the `createDocumentCoversheetOutline` method out of the `pdfOutlineService` and put it into this class as a private method. 

Although the method relates to outlines it doesn't belong with the rest of the outline behaviour which is why you've got two versions of the constructor.